gulp-ng-config.js

```diff
@@ -30,10 +32,22 @@ function gulpNgConfig (moduleName, configuration) {
         jsonObj,
         wrapTemplate;
 
-    try {
-      jsonObj = JSON.parse(file.contents.toString('utf8'));
-    } catch (e) {
-      this.emit('error', new PluginError(PLUGIN_NAME, 'invaild JSON file provided'));
+    if (_.endsWith(file.path, 'yml')) {
```

Comment:
Let's also support the alias `yaml` for the file extension and the `configuration.parser`